.github/workflows/rm-prime_2.8_and_2.9.yaml

```diff
@@ -0,0 +1,59 @@
+# This workflow calls the master E2E workflow with custom variables
+name: UI-RM_PRIME_2.8_AND_2.9
```

Comment:
what about `UI-RM_Prime`?

Reply:
It will indicate UI test cases for Rancher Manager Prime.

Reply:
No, this workflow will consist of Prime RC's/Alpha's as well released one too. Can you please suggest any other name?

Reply:
How about: `UI-RM_PRIME_TESTING` ?

Reply:
How about this: `UI-RM_Prime_Alpha_RC`

Reply:
Updated with new suggested names.